pages/checkout/_checkout.html

```diff
@@ -33,6 +33,7 @@ <h5 class="margin-top-none text-center no-anchor">
               <li class="check-list-disabled" id="subscription-addon-1"><span class="help-text light" title="Our paid support program where you can contact us via email or private Slack. 2 business days SLA." data-toggle="tooltip">Professional Support</span></li>
               <li class="check-list-disabled" id="enterprise-support"><span class="help-text light" title="Our Enterprise Support includes support via email, private Slack, and phone video, all with a same-business-day response time commitment." data-toggle="tooltip">Enterprise Support</span></li>
               <li class="check-list-disabled" id="subscription-addon-2"><span class="help-text light" title="An opinionated, best-practices, end-to-end tech stack that we customize to your needs, deploy into your AWS accounts, and give you 100% of the code—all in about one day." data-toggle="tooltip">Reference Architecture</span></li>
+              <li class="check-list-disabled" id="subscription-addon-3"><span class="help-text light" title="Learn how to achieve and maintain compliance with each of the CIS AWS Foundations Benchmark recommendations" data-toggle="tooltip">CIS Compliance</span></li>
```

Comment:
"Learn how" sounds like we provide a guide and the user has to do lots of stuff... Should we rephrase to make it clear we provide code/modules that are already compliant with those benchmarks?

Reply:
Rephrased to:
> Use infrastructure as code to achieve and maintain compliance with each of the CIS AWS Foundations Benchmark recommendations